SUMMARY.md

```diff
@@ -11,6 +11,7 @@
     * [Authentication](up-and-running/auth/README.md)
       * [Open Authentication](up-and-running/auth/open.md)
       * [Github Enterprise Integration](up-and-running/auth/github.md)
+  * [Uninstalling npm Enterprise](up-and-running/uninstalling.md)
```

Comment:
I certainly don't want to hide the info, but I'll buy your argument for a dollar. Change incoming.